dbt_platform_helper/providers/io.py

```diff
@@ -7,6 +7,9 @@ class ClickIOProvider:
     def warn(self, message: str):
         click.secho(message, fg="magenta")
 
+    def debug(self, message: str):
```

Comment:
Out of interest, can we set the log level with configuration and/or a command option?

Reply:
Currently no, I thought adding in this level could be the first step. 
My thinking is we could add a check initially the check if `DEBUG=True` and if it isn't this function wouldn't call the click function.